komodo/release_transpiler.py

```diff
@@ -32,9 +44,25 @@ def build_matrix_file(release_base, release_folder, builtins):
     write_to_file(compiled, f"{release_base}.yml", False)
 
 
-def _build(packages, py_ver, rhel_ver):
+def get_py_coords(release_base: str, release_folder: str) -> list:
+    """Get python versions of release files inside a given release_folder."""
+    filenames_with_prefix = sorted(
+        [
+            filename
+            for filename in os.listdir(release_folder)
+            if filename.startswith(release_base)
+        ]
+    )
+    len_release_base = len(release_base + "-")
+    py_coords = [filename[len_release_base:-4] for filename in filenames_with_prefix]
+    return py_coords
+
+
+def _build(packages: OrderedDict, rhel_ver: str, py_ver: str) -> dict:
+    """Consolidate the packages for a given python version into a dictionary."""
```

Comment:
The function also takes an `rhel` version, so maybe the docstring should reflect that.

Did you try just passing in arbitrary coordinates here? I still think it should be possible to make this function agnostic to the matrix. (E.g. have it consume `coords` instead of `rhel_ver` and `py_ver` and loop over its keys.)

Reply:
Okay, if what you have does what's needed then :+1: